x/emissions/migrations/v2/migrations.go

```diff
@@ -3,16 +3,27 @@ package v2
 import (
 	"fmt"
 
+	"cosmossdk.io/collections"
+	emissionsTypes "github.com/allora-network/allora-chain/x/emissions/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// An example MigrateStore function that might be run in the migration
-// handler.
-func MigrateStore(ctx sdk.Context) error {
-	ctx.Logger().Error(fmt.Sprintf(
+// for upgrade v0.2.8, delete the contents of
+// the stakeRemoval and delegateStakeRemoval
+// key value stores
+func MigrateStore(
+	ctx sdk.Context,
+	stakeRemoval *collections.Map[collections.Pair[uint64, string], emissionsTypes.StakePlacement],
+	delegateStakeRemoval *collections.Map[collections.Triple[uint64, string, string], emissionsTypes.DelegateStakePlacement],
+) error {
+	ctx.Logger().Warn(fmt.Sprintf(
 		"###################################################" +
 			"### MIGRATING STORE FROM VERSION 1 TO VERSION 2 ###" +
 			"###################################################",
 	))
-	return nil
+	err := stakeRemoval.Clear(ctx, nil)
+	if err != nil {
+		return err
+	}
+	return delegateStakeRemoval.Clear(ctx, nil)
 }
```

Comment:
that's correct, we're changing from one data structure to another for the stake removals

given that we're performing this on a testnet with little to no activity, it's simpler to just delete the removals and tell people to restart them rather than actually migrate them. It's 99% likely that there will be no removals in the state machine anyways

Reply:
(if we were doing this on mainnet, I wouldn't be so cavalier, and instead would copy the removals)